src/api/ApiClient.ts

```diff
@@ -49,10 +49,10 @@ class ApiClient {
   debug = (title: string, content: object) => {
     if (this.debug_mode && (title || content)) {
       if (title) {
-        console.log(`=== DEBUG: ${title} ===========================`)
+        // console.log(`=== DEBUG: ${title} ===========================`)
```

Comment:
Are they still needed?

Reply:
For debugging purposes it is nice to have